### PR TITLE
Simple Status Implementation

### PR DESCRIPTION
**Pull request by TayGov**
# Description

This PR includes the following proposed change(s):

Add service
Add Implementation
Add Test

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

## How Has This Been Tested?

Unit tests

## Does the change impact or break the Docker build?

- [ ] Yes
- [X] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [X] No

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/libs/efiling-commons/src/main/java/ca/bc/gov/open/jag/efilingcommons/service/EfilingStatusService.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package ca.bc.gov.open.jag.efilingcommons.service;
+
+import ca.bc.gov.open.jag.efilingcommons.model.FilePackage;
+
+import javax.xml.datatype.DatatypeConfigurationException;
+import java.math.BigDecimal;
+
+public interface EfilingStatusService {
+
+    FilePackage findStatusByPackage(BigDecimal clientId, BigDecimal packageNo) throws DatatypeConfigurationException;
```
Comment (alexjoybc) [inline src/backend/libs/efiling-commons/src/main/java/ca/bc/gov/open/jag/efilingcommons/service/EfilingStatusService.java:10]: suggestion `findStatusByPackageId`

File: src/backend/libs/efiling-cso-client/src/main/java/ca/bc/gov/open/jag/efilingcsoclient/CsoStatusServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        this.filingStatusFacadeBean = filingStatusFacadeBean;
+        this.filePackageMapper = filePackageMapper;
+    }
+
+    @Override
+    public FilePackage findStatusByPackage(BigDecimal clientId, BigDecimal packageNo) {
+        try {
+
+            FilingStatus filingStatus = filingStatusFacadeBean.findStatusBySearchCriteria(clientId, null, null, null, null, null, packageNo, null, null, null, null, null, BigDecimal.ONE, null);
+            if (filingStatus.getFilePackages().isEmpty())  throw new EfilingStatusServiceException("No records for that package");
```
Comment (alexjoybc) [inline src/backend/libs/efiling-cso-client/src/main/java/ca/bc/gov/open/jag/efilingcsoclient/CsoStatusServiceImpl.java:30]: should it return null ?

**Review by alexjoybc — CHANGES_REQUESTED** [review]

**Review by alexjoybc — APPROVED** [review]
looks good thanks! @TayGov

File: src/backend/libs/efiling-commons/src/main/java/ca/bc/gov/open/jag/efilingcommons/service/EfilingStatusService.java (hunk 1 above)
Comment (TayGov) [reply src/backend/libs/efiling-commons/src/main/java/ca/bc/gov/open/jag/efilingcommons/service/EfilingStatusService.java]: 👍

**Pull request merged**